bin/build-checks.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+/* eslint-disable global-require, no-console */
+
+const chalk = require('chalk');
+
+require('babel-register');
+const config = require('config');
+
+const appName = config.get('appName');
+
+// Bail if appName isn't set unless explicitly enabled.
+if (!appName && parseInt(process.env.ADDONS_FRONTEND_BUILD_ALL, 10) !== 1) {
```

Comment:
I think `process.env.ADDONS_FRONTEND_BUILD_ALL === '1'` would be clearer. Is there a reason you need to convert to int?